db/schema.rb

```diff
@@ -115,7 +115,13 @@
     t.datetime 'updated_at', precision: 6, null: false
     t.boolean 'use_billing', default: false
     t.bigint 'customer_id'
+    t.bigint 'delivery_id'
+    t.bigint 'credit_card_id'
+    t.string 'number'
+    t.integer 'status'
```

Comment:
```ruby
t.integer 'status', default: 0
```
Otherwise, you should define it each time after creation, which is not good approach.

Reply:
Fixed as mentioned above